Dart/src/com/jetbrains/lang/dart/validation/fixes/DartServerFixIntention.java

```diff
@@ -180,9 +186,24 @@ public boolean startInWriteAction() {
   @Override
   public void invoke(@NotNull final Project project, final Editor editor, final PsiFile file) throws IncorrectOperationException {
     final SourceFileEdit fileEdit = myChange.getEdits().get(0);
+    final String filePath = fileEdit.getFile();
     final SourceEdit sourceEdit = fileEdit.getEdits().get(0);
 
-    final VirtualFile virtualFile = LocalFileSystem.getInstance().findFileByPath(FileUtil.toSystemIndependentName(fileEdit.getFile()));
+    // Create the file if it does not exist.
+    if (fileEdit.getFileStamp() == -1) {
+      try {
+        final String directoryPath = VfsUtil.getParentDir(filePath);
```

Comment:
Will fail on Windows because of different slashes. Slash tolerant methods: PathUtil.getParentPath, PathUtil.getFileName. Fixed